tests/utils/constants/http_constants.go

```diff
@@ -0,0 +1,21 @@
+package constants
+
+// HTTP response code
+const (
+	HTTPOK        = 200
```

Comment:
Is it used?
import "net/http"
it includes all the statuses , i.e. http.StatusOK